source/TakeNotes.rdef

```diff
@@ -15,20 +15,20 @@
  *
  * Description: TODO
  */
- 
+
 
 resource app_signature "application/x-vnd.ccc-TakeNotes";
 
 resource app_version {
 	major  = 1,
-	middle = 0,
+	middle = 5,
 	minor  = 0,
 
-	variety = B_APPV_BETA,
+	variety = B_APPV_FINAL,
 	internal = 0,
 
 	short_info = "TakeNotes",
-	long_info = "TakeNotes ©2009 Ilio Catallo, Stefano Celentano, Eleonora Ciceri"
+	long_info = "TakeNotes ©2021 Ilio Catallo, Stefano Celentano, Eleonora Ciceri, Florian Thaler"
```

Comment:
The long_info shouldn't be used for copyrights/credits, but a description of the app in one short sentence.